src/components/home/Post.tsx

```diff
@@ -162,20 +163,37 @@ export const Post: React.FC<PostProps> = React.memo(({post, style}) => {
     return () => ro.disconnect();
   }, [nameRef.current, messageRef.current]);
 
-  
+  /**
+   * Displays the details of this post in a popup.
+   */
+  const popup = (): void => {
+    setShowPopup(true);
+  };
+
+  /**
+   * Closes the shown popup, if any.
+   */
+   const closePopup = (): void => {
+    setShowPopup(false);
+  }
+
```

Comment:
Why don't you just use anonymous functions when needing them since all it does it set a state true/false

no need to scroll up to see what it does when these functions are called

Reply:
I think these functions already originally had longer values but now they don't anymore and I just forgot to turn them into anonymous functions